docs/guide/input.rst

```diff
@@ -0,0 +1,22 @@
+.. _input:
```

Comment:
this file should be included in some ``toctree`` directive.

Reply:
I have included the input info in the file howtorun.rst now so I think input.rst can be removed